### PR TITLE
[Fix] [WRS-1557] Handle scenario when name is not available in relative path

### PR DESCRIPTION
**Pull request by abdelhalimkhouas**
This PR formats the relative path if the the name of the folder is not included in the `relativePath`

## Related tickets

-   [WRS-1557](https://chilipublishintranet.atlassian.net/browse/WRS-1557)

## Screenshots

[WRS-1557]: https://chilipublishintranet.atlassian.net/browse/WRS-1557?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,21 +15,39 @@
...
                 var queryString = window.location.search;
                 var urlParams = new URLSearchParams(queryString);
+                // the engineVersion should either be changed here or in the URL (PR number or main)
+                var engineVersion = urlParams.get('engine') ?? 'main';
+                var engineCommitSha = urlParams.get('engineCommitSha');
+                // the engineCommitSha can be changed here or in the URL
+                engineCommitSha =
+                    engineCommitSha && engineCommitSha !== ''
+                        ? engineCommitSha
+                        : '857211e434199c0428234f085c06c6d26bb2d098';
```
Comment (psamusev) [inline index.html:25]: I don't think that we need specify default value for `hash`

File: src/components/itemBrowser/ItemBrowser.tsx [hunk 2]
```diff
@@ -204,6 +204,13 @@ function ItemBrowser<
 
     const getKey = useCallback((str: string, idx: number) => encodeURI(`${str},${idx}`), []);
 
+    const formatRelativePath = useCallback((item: T) => {
```
Comment (psamusev) [inline src/components/itemBrowser/ItemBrowser.tsx:207]: You're not using it in any `deps` array, so useCallback is useless

Comment (abdelhalimkhouas) [reply src/components/itemBrowser/ItemBrowser.tsx]: done 4751e1dbcb2c0eb8516b8116398baaf62856fc94

File: index.html (hunk 1 above)
Comment (abdelhalimkhouas) [reply index.html]: done 8d1357c6152e99f6d17c62b11db0273881173bdb

**Review by alexandraFlavia9 — APPROVED** [review]

**Review by psamusev — APPROVED** [review]

**Pull request merged**